README.md

```diff
@@ -192,6 +192,9 @@ The CR includes the following parameters:
 * `retrycount` - number of times to retry the fence agent in case of failure. The default is 5.
 * `retryinterval` - interval between retries in seconds. The default is "5s".
 * `timeout` - timeout for the fence agent in seconds. The default is "60s".
+* `remediationStrategy` - either `ResourceDeletion` or `OutOfServiceTaint`:
+    * `ResourceDeletion`: This remediation strategy removes the pods on the node.
```

Comment:
Maybe it's me only, but why don't call things how they are called already? 🤔 
Here: we do not `remove` anything, we `delete` them... that's the actual method name we use. That's the `D` in CRUD. 🤷🏼‍♂️ (Same below)

(and I don't block on this, just a general thought, which I had more often at various places)

Reply:
/lgtm

in case you want to change...
/hold

Reply:
> why don't we call things how they are called already

for me, I think it's just an automatically mechanism to avoid repetitions in written documents, but indeed when explaining something this habit can be limited :)